autoscraper/__init__.py

```diff
@@ -1 +1 @@
-from autoscraper.auto_scraper import AutoScraper
+from .auto_scraper import AutoScraper
```

Comment:
Absolute imports are preferred in pep8. 
https://www.python.org/dev/peps/pep-0008/#imports